packages/tar.rb

```diff
@@ -3,24 +3,13 @@
 class Tar < Package
   description 'GNU Tar provides the ability to create tar archives, as well as various other kinds of manipulation.'
   homepage 'https://www.gnu.org/software/tar/'
-  version '1.33'
+  version '1.34'
   license 'BSD'
   compatibility 'all'
-  source_url 'https://ftpmirror.gnu.org/tar/tar-1.33.tar.xz'
-  source_sha256 '66a8344b1dc83a411d311bd1547e0176e56cc311f28ee94a30f84dafb3d9067e'
+  source_url 'https://ftpmirror.gnu.org/gnu/tar/tar-1.34.tar.gz'
```

Comment:
The url should work without the `gnu` also.

Reply:
What is wrong with the `.tar.xz` extension?  Should be a smaller tarball with better compression.

Reply:
Since this file is auto-generated by my [update checker](https://github.com/supechicken/crew-package-updater), some of the sources may be changed